### ISSUE TITLE
Bug: WP Offload Media Does not detect Banglore Sever of DigitalOcean

### ISSUE THREAD

**Issue by aeyvam** [deliciousbrains/wp-amazon-s3-and-cloudfront#633]
I've encountered a critical bug with the WP Offload Media plugin, where it fails to detect the Bangalore server on DigitalOcean. While all other servers display the spaces bucket, the one created in Bangalore does not appear. This issue persists across multiple accounts, indicating it's not an isolated incident. 
now it is impossible to offload media on DigitalOcean's Bangalore server. Please prioritize this bug fix as soon as possible.

**Comment by eriktorsner** [comment]
Hi, 

Thanks for reporting this. Seems that we missed that Spaces had become available in Bangalore when we did our last release back in August.

We have raised an issue to take care of this as soon as possible, BLR1 should become available as a selectable region in the next release of WP Offload Media.

In the meantime, you can workaround this issue by implementing a filter in a suitable place, I believe this will do the trick:

```
add_filter( 'as3cf_do_get_regions', "do_get_regions", 10, 1 );
function do_get_regions( $regions ) {
	if ( ! isset( $regions['blr1'] ) ) {
		$regions['blr1'] = 'Bangalore';
	}

	return $regions;
}
```
Please let us know how this works out for you. Again, thanks for reporting this, and sorry for the inconvenience.

**Comment by ianmjones** [comment]
This was addressed in v3.2.7.

Thanks for the bug report.
